packages/protocol/test/identity/attestations.ts

```diff
@@ -89,17 +95,62 @@ contract('Attestations', (accounts: string[]) => {
     return accounts[nonIssuerIndex]
   }
 
+  const getValidatingKeyAddress = (address: string) => {
+    const pKey = accountPrivateKeys[accounts.indexOf(address)]
+    const aKey = Buffer.from(pKey.slice(2), 'hex')
+    aKey.write((aKey[0] + 2).toString(16))
+    return privateKeyToAddress('0x' + aKey.toString('hex'))
+  }
+
+  const getAttestationKey = (address: string) => {
+    const pKey = accountPrivateKeys[accounts.indexOf(address)]
+    const aKey = Buffer.from(pKey.slice(2), 'hex')
+    aKey.write((aKey[0] + 1).toString(16))
+    return '0x' + aKey.toString('hex')
+  }
+
+  const unlockAttestationKey = async (address: string) => {
+    const attestationKey = getAttestationKey(address)
+    const attestationKeyAddress = privateKeyToAddress(attestationKey)
+    // @ts-ignore
+    await web3.eth.personal.importRawKey(attestationKey, 'passphrase')
+    await web3.eth.personal.unlockAccount(attestationKeyAddress, 'passphrase', 1000000)
+    return attestationKeyAddress
+  }
+
+  const registerAttestationKey = async (address: string) => {
+    const attestationKey = getAttestationKey(address)
+    const attestationKeyAddress = privateKeyToAddress(attestationKey)
+    const attestationSig = await getParsedSignatureOfAddress(web3, address, attestationKeyAddress)
+
+    return attestations.setAttestationKeyAddress(
+      attestationKeyAddress,
+      attestationSig.v,
+      attestationSig.r,
+      attestationSig.s,
+      { from: address }
+    )
+  }
+
   beforeEach(async () => {
     mockStableToken = await MockStableToken.new()
     otherMockStableToken = await MockStableToken.new()
     attestations = await Attestations.new()
     random = await Random.new()
     mockValidators = await MockValidators.new()
-    await Promise.all(accounts.map((account) => mockValidators.addValidator(account)))
+    await Promise.all(
+      accounts.map((account) => mockValidators.addValidator(getValidatingKeyAddress(account)))
```

Comment:
We should be adding the locked gold account here an not the authorized validator

Reply:
So we currently set it this way here just so that we can access the list of validators from `MockValidators`. I did that so that we test that the `Attestations` contract correctly calls `LockedGold` to retrieve the LockedGold account.